.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,6 +4,8 @@ Please review the [guidelines for contributing](http://solidity.readthedocs.io/e
 
 Please also note that this project is released with a [Contributor Code of Conduct](CONDUCT.md). By participating in this project you agree to abide by its terms.
 
+Please delete everything in this template apart from the checklist itself.
+
```

Comment:
@chriseth 
A better idea is to put it inside html comments. This way even if they don't delete them, they won't appear to be visible in the final issue. Check out the issue templates we got in raiden. https://raw.githubusercontent.com/raiden-network/raiden/master/.github/ISSUE_TEMPLATE/bug_report.md